Makefile

```diff
@@ -61,17 +69,22 @@ clean : clean-app
 clean-app:
 	$(GOCLEAN) $(API_PATH)
 	$(GOCLEAN) $(PROXY_PATH)
+	$(GOCLEAN) $(PROXY_CARTHAGENET_PATH)
 	$(GOCLEAN) $(SNAPSHOT_PATH)
 	$(GOCLEAN) $(METRICS_CLEANER_PATH)
 	rm -f $(API_BIN)
 	rm -f $(PROXY_BIN)
+	rm -f $(PROXY_CARTHAGENET_BIN)
 	rm -f $(SNAPSHOT_BIN)
 	rm -f ${METRICS_CLEANER_BIN}
 build-docker: build-unix
 	docker-compose build
 run:
 	docker-compose up -d postgres node $(API) $(PROXY)
 	cd web && yarn start
+run-dev:
+	docker-compose up -d postgres node node-rolling $(API) $(PROXY) $(PROXY_CARTHAGENET)
```

Comment:
if you want to run all the stack, you could do `docker-compose up -d` instead. :)

Reply:
Next to our conversation I resolv this comment